README.md

```diff
@@ -50,6 +50,26 @@ python setup.py install
 ```
 
 
+## Docker
+
+Build docker image
+```bash
+docker build -t px-ldap-replace .
+```
+
+Prepare config files
+```bash
+mkdir .config
+vim .config/px-ldap-replace.yaml
+vim .config/mails.txt
```

Comment:
`mails.txt` is just an example for the `limit_file` parameter.
It does not need to be set.